Makefile

```diff
@@ -43,10 +45,22 @@ clean:
 
 TAG?=$(shell git rev-parse HEAD)
 
-images = $(shell ls build/docker/*.Dockerfile | sed 's/.*\/\(.\+\)\.Dockerfile/\1/')
+images = $(shell ls build/docker/*.Dockerfile | grep -v clearlinux-base | sed 's/.*\/\(.\+\)\.Dockerfile/\1/')
 
+clearlinux-base:
+	@test -d mod && (echo "Not clean workspace detected, please remove the directory 'mod'. Exiting..."; exit 1) || true
+	@mkdir mod
+	@$(SUDO) mount --bind $(shell $(GO) env GOPATH)/pkg/mod mod
```

Comment:
why bind mount, why not just symlink?

Reply:
With a symlink I get
```
Step 5/5 : COPY mod /go/pkg/mod
COPY failed: stat /home/rojkov/dockerstorage/tmp/docker-builder697108168/home/rojkov/go/pkg/mod: no such file or directory
```

Reply:
What about 
`COPY mod/* /go/pkg/mod/`

Reply:
It doesn't matter. `COPY` copies files from the container's context, not from the repo's directory.